netlify.toml

```diff
@@ -239,3 +239,10 @@ status = 302
 [[redirects]]
 from = "/test-toolkit/"
 to = "/test-toolkit/latest/"
+
+########################################################################################################################
+# Bonita Central redirects
+########################################################################################################################
+[[redirects]]
+from = "/central/"
+to = "/central/latest/"
```

Comment:
**question:** did you try this redirect (both with the tests content and the actual content)? the new version is 1.0, see https://github.com/bonitasoft/bonita-central-doc/blob/db2f557b43ed1ed8714f35e5d5403c02c2b409c1/antora.yml#L1-L4

So IMHO, the 302 redirect from latest to 1.0 is missing. This is required until we implement #419